frontend/src/components/associations/marketplace/ProductsAdministration.tsx

```diff
@@ -1,12 +1,44 @@
 import React from "react";
 import { PageTitle } from "../../utils/PageTitle";
+import { Pagination } from "../../utils/Pagination";
+import { api } from "../../../services/apiService";
+import Row from "react-bootstrap/Row";
+import { Product } from "./common/Product";
+import { Link } from "react-router-dom";
 
 export const AssociationMarketplaceProductAdministration = ({
     association,
 }) => {
+    let marketplaceId = association.id;
     return (
         <>
             <PageTitle>Produits</PageTitle>
+            <Row>
+                <Pagination
+                    apiKey={["products.list", marketplaceId, { page_size: 8 }]}
+                    apiMethod={api.products.list}
+                    render={(products, controlbar) => (
+                        <>
+                            <Row>
+                                {products.map((product) => (
+                                    <Product
+                                        product={product}
+                                        key={product.id}
+                                        additionalContent={
+                                            <Link
+                                                to={`/associations/${marketplaceId}/marketplace/products/${product.id}/edit`}
```

Comment:
please rename the URL to `modifier` or something
Actually, `products` should be `produit`

Reply:
And even `marketplace` should be translated

Reply:
`magasin` is great